### PR TITLE
SSO: remove Jetpack.com redirect during login process

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

﻿This was added in #15140. Removing it means making the log in process a bit faster.

#### Jetpack product discussion

* Internal reference: p55Cj4-1bO-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a site with SSO enabled, log out and log back in.
* You should be redirected to WordPress.com and then back to your site, without going through jetpack.com.

### CODE REVIEW

**Comment by josephscott** [comment]
+1 to this, reducing the number of redirects involved in SSO will get users to the actual login page faster.

**Comment by sdixon194** [comment]
This works for me, but will hold off approving pending the discussion here: p55Cj4-1bO-p2#comment-2781

**Review by kraftbj — APPROVED** [review]
I'll approve it.

The summary of the conversation is that we'd lose the ability to dynamically change the URL of the redirect for older versions/without a Jetpack release if that URL ever changes.

Since the wp.com login URL probably won't and, even if it does, we'll have to figure out a clean redirect on the WP.com side, so let's do this.

**Pull request merged**
